doc/manual/src/store/path.md

```diff
@@ -0,0 +1,128 @@
+# Store Path
+
+Nix implements references to [store objects](./index.md#store-object) as *store paths*.
+
+Think of a store path as an [opaque], [unique identifier]:
+The only way to obtain store path is by adding or building store objects.
```

Comment:
Say something about "hard to spoof" / "hard to forge?" That is why we care that they can only be produced this way.

Reply:
That would need some explanation on why exactly being hard to forge matters. I think this can go into the digest section, which will now end up in a separate PR.